src/main/java/seedu/us/among/model/endpoint/Address.java

```diff
@@ -16,7 +16,9 @@
 public class Address {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "URL has to be a valid URL that does not contain any illegal characters. E.g. https://localhost:3000";
+            "URL provided has to be valid and does not contain any illegal characters.\nFor example:\n"
+                    + "https://localhost:3000\n"
+                    + "https://api.data.gov.sg/v1/environment/air-temperature";
```

Comment:
maybe can add a OR between the 2 examples, or can put 1. 2. ...

Reply:
ok